inc/class-core-sitemaps-provider.php

```diff
@@ -111,6 +111,27 @@ public function get_url_list( $page_num ) {
 			);
 		}
 
+		// Show a URL for the homepage in the pages sitemap if the reading settings are set to display latest posts.
```

Comment:
We need to add a check to make sure this is on the first page only. Otherwise, the homepage URL will show up on every sitemap where `$post_type === 'page'`.

Nitpick, but I'd prefer to see this happen before the rest of the URLs are added, so that the homepage URL is the first URL in the list, rather than the last.